### PR TITLE
mimic: PGLog.cc: saw valgrind issues while accessing complete_to->version

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/26909

### CODE REVIEW

**Comment by yuriw** [comment]
[wip-yuri-testing-2018-09-02-2228-mimic](https://trello.com/c/NY950iNm/384-wip-yuri-testing-2018-09-02-2228-mimic)

**Comment by xiexingguo** [comment]
This should include https://github.com/ceph/ceph/pull/23546 too

**Comment by smithfarm** [comment]
@xiexingguo Thanks; done.

**Comment by smithfarm** [comment]
@yuriw Marking DNM because this needs to be retested. Please remove the DNM when you re-test.

File: src/osd/PrimaryLogPG.cc [hunk 1]
```diff
@@ -11437,6 +11437,8 @@ void PrimaryLogPG::_applied_recovered_object_replica()
 void PrimaryLogPG::recover_got(hobject_t oid, eversion_t v)
 {
   dout(10) << "got missing " << oid << " v " << v << dendl;
+  dout(10) << __func__ << " complete_to "
+           << pg_log.get_log().complete_to->version << dendl;
```
Comment (xiexingguo) [inline src/osd/PrimaryLogPG.cc]: On second thought I think it is (also) not safe to directly log the content of  the **complete_to** iter here. I post yet another fix at https://github.com/ceph/ceph/pull/23894

**Comment by smithfarm** [comment]
Closing in favor of https://github.com/ceph/ceph/pull/23403

**Pull request closed without merging**